intellij-plugin-structure/structure-base/src/main/kotlin/com/jetbrains/plugin/structure/jar/PluginJar.kt

```diff
@@ -21,6 +23,8 @@ val PLUGIN_XML_RESOURCE_PATH = META_INF + File.separator + PLUGIN_XML
 private val THIRD_PARTY_LIBRARIES_FILE_NAME = "dependencies.json"
 
 class PluginJar(private val jarPath: Path, private val jarFileSystemProvider: JarFileSystemProvider = DefaultJarFileSystemProvider()): AutoCloseable {
+  private val LOG: Logger = LoggerFactory.getLogger(PluginJar::class.java)
```

Comment:
AFAIU LOG instance could be top-level to avoid instantiation for every instance

Reply:
Fixed.

By the way, [SLF4J FAQ](https://www.slf4j.org/faq.html#declared_static) says neither approach is better than the other one.

Reply:
we have this policy for loggers in IJ for a reason